src/main/kotlin/com/team4099/robot2023/commands/drivetrain/TargetAngleCommand.kt

```diff
@@ -75,29 +79,35 @@ class TargetAngleCommand(
       thetakD.initDefault(DrivetrainConstants.PID.TELEOP_ALIGN_PID_KD)
 
       thetaPID =
-        PIDController(
+        ProfiledPIDController(
           DrivetrainConstants.PID.TELEOP_ALIGN_PID_KP,
           DrivetrainConstants.PID.TELEOP_ALIGN_PID_KI,
-          DrivetrainConstants.PID.TELEOP_ALIGN_PID_KD
+          DrivetrainConstants.PID.TELEOP_ALIGN_PID_KD,
+          TrapezoidProfile.Constraints(
+            DrivetrainConstants.STEERING_VEL_MAX, DrivetrainConstants.STEERING_ACCEL_MAX
+          )
         )
     } else {
       thetakP.initDefault(DrivetrainConstants.PID.SIM_AUTO_THETA_PID_KP)
       thetakI.initDefault(DrivetrainConstants.PID.SIM_AUTO_THETA_PID_KI)
       thetakD.initDefault(DrivetrainConstants.PID.SIM_AUTO_THETA_PID_KD)
 
       thetaPID =
-        PIDController(
+        ProfiledPIDController(
           DrivetrainConstants.PID.SIM_AUTO_THETA_PID_KP,
           DrivetrainConstants.PID.SIM_AUTO_THETA_PID_KI,
-          DrivetrainConstants.PID.SIM_AUTO_THETA_PID_KD
+          DrivetrainConstants.PID.SIM_AUTO_THETA_PID_KD,
+          TrapezoidProfile.Constraints(
+            DrivetrainConstants.STEERING_VEL_MAX, DrivetrainConstants.STEERING_ACCEL_MAX
+          )
         )
     }
 
     thetaPID.enableContinuousInput(-PI.radians, PI.radians)
   }
 
   override fun initialize() {
-    thetaPID.reset() // maybe do first for x?
+    thetaPID.reset(0.degrees) // maybe do first for x?
```

Comment:
why 0 degrees?

Reply:
what would we reset it to then? the current gyro angle?

Reply:
you're not resetting the angle, you're resetting the error it sees. we don't want error from the previous path to carry on, but rather recalculate in our execute function